go.mod

```diff
@@ -2,7 +2,7 @@ module github.com/rancher/eks-operator
 
 go 1.22.0
 
-toolchain go1.22.5
+toolchain go1.22.7
 
 replace k8s.io/client-go => k8s.io/client-go v0.28.6
 
```

Comment:
I believe the reason why we didn't get any dependabot PRs for 2.8 is because it's still using the AWS SDK for Go (v1) which is in maintenance mode and is only receiving critical bug fixes and security updates. In addition to that, this version of the SDK (v1) is not using the same stdlib dependencies that the newer version of the SDK (v2) uses.

I think we should consider bumping this to its latest version but such a change should probably be done separately and be followed by sufficient testing.